gobblin-utility/src/main/java/gobblin/util/AvroUtils.java

```diff
@@ -80,6 +81,13 @@
 
   private static final String AVRO_SUFFIX = ".avro";
 
+  public static class AvroPathFilter implements PathFilter {
```

Comment:
No, it was until I realized it broke the input format. I figures I'd leave it there for the future.